pkg/lint/rules/template.go

```diff
@@ -45,7 +45,7 @@ var (
 )
 
 // Templates lints the templates in the Linter.
-func Templates(linter *support.Linter, values map[string]interface{}, namespace string, strict bool) {
+func Templates(linter *support.Linter, values map[string]interface{}, namespace string, strict bool, skipSchemaValidation bool) {
```

Comment:
this probably needs a new method like (ToRenderValuesSkipSchemaValidation) to keep api backward compatibilty ?

Reply:
Correct 👍 we can not change the signature of public functions in Helm 3